tests/5.0/target/test_target_map_with_close_modifier.F90

```diff
@@ -0,0 +1,66 @@
+!===--- test_target_map_with_close_modifier.F90 -------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test checks for support of the close map-type-modifier on a map clause. The test maps several
+! different data types to device with tofrom map-type and then checks for expected updated values on
+! the host.
+! 
+!//===----------------------------------------------------------------------===//
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_map_with_close_modifier
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(test_close_modifier() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_close_modifier()
+    INTEGER, DIMENSION(N) :: a
+    INTEGER :: errors, i, scalar
+
+    TYPE memberT
+    INTEGER :: var
+    INTEGER, DIMENSION(N) :: b
+    ENDTYPE memberT
+
+    TYPE(memberT) :: member
+
+    scalar = 19
+    member%var = 1
+
+    errors = 0
+
+    DO i = 1, N
+       a(i) = i
+       member%b(i) = i
+    END DO
+
+    !$omp target map(close,tofrom : scalar, a, member)
+    scalar = 25
```

Comment:
Minor  nitpick, I would recommend using value of "scalar" to update value that way it can check if it was mapped to accurately.

Reply:
Then, change "scalar = 25" to "scalar = scalar + 25"? 
We can change other statements similarly. (We will have to change both C and Fortran versions.)

Reply:
yes, it would be a more complete test IMO.

Reply:
Updated both C and Fortran versions.
Tested with GCC 11.1, IBM XL 16.1.1-10, NVHPC 21.11, and LLVM 15.0, but the C version passes only when compiled by NVHPC 21.11. (Fortran version fails all.)